docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>To write this more compact, you could do:</p><p> </p><pre class="ql-syntax" spellcheck="false">specs = {
&nbsp; &nbsp; 'filter_scale': filter_scale,
&nbsp; &nbsp; 'dx_min': dx_min,
&nbsp; &nbsp; 'grid_type': gcm_filters.GridType.REGULAR_WITH_LAND_AREA_WEIGHTED,
&nbsp; &nbsp; 'grid_vars': {'area': dA, 'wet_mask': dWM}
}


filter_simple_fixed_factor_G = gcm_filters.Filter(filter_shape=gcm_filters.FilterShape.GAUSSIAN, **specs)
filter_simple_fixed_factor_T = gcm_filters.Filter(filter_shape=gcm_filters.FilterShape.TAPER, **specs)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>